src/lib/holocene/tab/tab.svelte

```diff
@@ -64,13 +64,13 @@
 
 <style lang="postcss">
   .tab {
-    @apply flex cursor-pointer items-center gap-1 whitespace-nowrap border-b-2 text-sm hover:border-b-2 hover:border-blue-700 md:text-base;
+    @apply flex cursor-pointer items-center gap-1 whitespace-nowrap border-b-2 text-sm outline-none hover:text-brand focus-visible:rounded-sm focus-visible:shadow-focus focus-visible:shadow-indigo-300 dark:text-secondary dark:hover:text-brand;
 
     border-color: transparent;
   }
 
   .tab.active {
-    @apply border-b-2 border-blue-700 font-medium text-blue-700;
+    @apply border-information font-medium;
```

Comment:
I think we should move `font-medium` to the base tab styles so that the font size doesn't change when it's active